pkg/build/pipelines/pecl/install.yaml

```diff
@@ -0,0 +1,19 @@
+name: Installs and enables a PHP PECL module.
+
+needs:
+  packages:
+    - automake
+    - busybox
+    - cmake
+    - php-8.1-dev
```

Comment:
In general I would prefer to not tie these to specific PHP versions, and instead have the user specify that a PHP version is present.  I think we can do that by depending on a `php-dev` virtual package, and requiring that the user install the php-dev package from the version stream they actually want in the build recipe.

Reply:
makes sense!